src/plugins/intel_cpu/src/config.cpp

```diff
@@ -411,6 +412,9 @@ void Config::readProperties(const ov::AnyMap& prop, const ModelType modelType) {
         if (!fcDynamicQuantizationGroupSizeSetExplicitly) {
             fcDynamicQuantizationGroupSize = 0;
         }
+        if (!kvCachePrecisionSetExplicitly) {
```

Comment:
why not set `kvCachePrecision` to `u8` here ? To make the `kvCachePrecision` compatible in ACL platform ? If so, it's better to left a comment here to explain this.

Reply:
Setting the `kvCachePrecision` in constructor should be more clearer and consistent with other properties such as `DynamicQuantizationGroup`, so my suggestion is to keep them in current style.